src/index.js

```diff
@@ -8,19 +8,41 @@ import styles from './styles.css';
 // =============================================================================
 function docsifyCopyCode(hook, vm) {
     hook.doneEach(function() {
-        const codeBlocks = Array.apply(null, document.querySelectorAll('pre[data-lang]'));
-
-        codeBlocks.forEach(function(element, i, obj) {
-            const button = document.createElement('button');
+        const targetElms = Array.apply(null, document.querySelectorAll('pre[data-lang]'));
+        const i18n = {
+            buttonText: 'Copy to clipboard',
+            errorText: 'Error',
+            successText: 'Copied'
+        };
 
-            button.appendChild(document.createTextNode('Click to copy'));
-            button.classList.add('docsify-copy-code-button');
+        // Update i18n strings based on options and location.href
+        if (vm.config.copyCode) {
+            Object.keys(i18n).forEach(key => {
+                const textValue = vm.config.copyCode[key];
 
-            if (vm.config.themeColor) {
-                button.style.background = vm.config.themeColor;
-            }
+                if (typeof textValue === 'string') {
+                    i18n[key] = textValue;
+                }
+                else if (typeof textValue === 'object') {
+                    const hrefMatch = Object.keys(textValue).filter(key => location.href.indexOf(key) > -1)[0];
```

Comment:
This works great - a slightly "cleaner" version would be `const hrefMatch = Object.keys(textValue).find(key => location.href.includes(key))`. Not that the performance on this small set will make any difference, but the `find` will then exit early when it matches instead of the `filter` looping through everything and then plucking the first item in the result. What do you think?

Reply:
Unfortunately, [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) isn't supported by IE and docsify supports IE10+.

Another approach is to use [Array.prototype.some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) which, although it returns a boolean, provides the index of a matching item available and also exits once a matching item is found.

I agree it's not a big deal given the small set we're working with. That said, It's an easy enough change to make.

Reply:
All set! See [latest commit](https://github.com/jperasmus/docsify-copy-code/pull/14/commits/8ab1e4a0014298ed1253004067a0ed22db8b55b2) for update.

Reply:
Okay, 100%. I was under the impression that our Rollup config might have polyfilled it, but took a quick look and didn't see it. Let's go with this approach then.

Reply:
Excellent! Do you want to merge and publish?